### PR TITLE
feat(release-please): allow configuring manifest releases

### PR DESCRIPTION
**Pull request by chingor13**
Towards #1531 

Upgrades release-please to 11.3.0

### CODE REVIEW

File: packages/release-please/src/release-please.ts [hunk 1]
```diff
@@ -52,6 +53,7 @@ interface BranchOptions {
   bumpMinorPreMajor?: boolean;
   path?: string;
   changelogPath?: string;
+  manifest?: boolean;
```
Comment (chingor13) [inline packages/release-please/src/release-please.ts:56]: I'm not particularly a fan of this configuration. I would prefer that the manifest releaser be a `ReleaseType`, but that's not available in release-please -- you have to use a specific factory method. If we know that `manifest` will be a release type, then I'd prefer handling this by adding a `manifest` enum option to `releaseType` and special case handling it to use the new factory until we clean up how to build a manifest PR.

**Comment by chingor13** [comment]
The alternate configuration I'd propose is to have `manifest` as a release type.

Example `.github/release-please.yaml`:
```yaml
releaseType: manifest
handleGHRelease: true
```

File: packages/release-please/test/release-please.ts [hunk 2]
```diff
@@ -263,6 +264,51 @@ describe('ReleasePleaseBot', () => {
...
+
+        await probot.receive({name: 'push', payload, id: 'abc123'});
+        requests.done();
+        assert(executed, 'should have executed the runner');
+      });
+
+      it('should handle GitHub releases, if configured', async () => {
+        let runnerExecuted = false;
+        let releaserExecuted = false;
+        Runner.manifest = async (manifest: Manifest) => {
```
Comment (bcoe) [inline packages/release-please/test/release-please.ts:291]: You could use sinon and replace this with something like:

```
const stub = sinon.stub(Runner, 'manifest');
sinon.assert.calledOnce(stub)
```

That way we get rid of the complained about `Manifest` being unused.

**Review by bcoe — APPROVED** [review]
Non-blocking feedback, that I think it would be a bit nicer to use `sinon.stub`.

File: packages/release-please/test/release-please.ts (hunk 2 above)
Comment (chingor13) [reply packages/release-please/test/release-please.ts:291]: Will do in a follow-up PR.

**Pull request merged**